.github/workflows/on-main-branch-commit.yml

```diff
@@ -36,10 +36,16 @@ jobs:
           test-case: button
         timeout-minutes: 15
 
+  deploy-button:
+    name: Deploy the button contracts to devnet and trigger services update
+    needs: run-e2e-button-contract-tests
```

Comment:
```suggestion
    needs: [run-e2e-button-contracts-tests]
```

Reply:
isn't this syntax for arrays?

Reply:
Yep is it, this should be array with one element like suggested change.

Reply:
ok